src/compiler/tsc.ts

```diff
@@ -675,7 +677,22 @@ namespace ts {
             usageText += getParamType(option);
 
             usageColumn.push(usageText);
-            descriptionColumn.push(getDiagnosticText(option.description));
+            let description: string;
+
+            if (option.name === "lib") {

```

Comment:
Surely you could be more general than this and check for if you have a `CommandLineOptionOfListType`